queries.sql

```diff
@@ -51,3 +51,17 @@ SELECT *FROM animals;
  SELECT animals.name FROM animals JOIN owners ON animals.owner_id = owners.id JOIN species ON animals.species_id = species.id WHERE owners.full_name = 'Jennifer Orwell' AND species.name = 'Digimon';
  SELECT animals.name FROM animals JOIN owners ON animals.owner_id = owners.id WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts = 0;
  SELECT owners.full_name FROM animals JOIN owners ON animals.owner_id = owners.id GROUP BY owners.full_name ORDER BY COUNT(animals.id) DESC LIMIT 1;
+
+
+
+--  JOIN TABLES QUERY
+
+SELECT animals.name FROM visits  JOIN vets ON visits.vets_id = vets.id  JOIN animals ON visits.animals_id = animals.id WHERE vets.name = 'William Tatcher' ORDER BY date DESC LIMIT 1;
+SELECT COUNT(date) FROM visits JOIN vets ON vets_id = vets.id WHERE vets.name = 'Stephanie Mendez';
+SELECT vets.name, species.name FROM vets LEFT JOIN specializations ON vets.id = vets_id LEFT JOIN species ON species.id = species_id;
+SELECT animals.name,visits.date FROM visits  JOIN vets ON visits.vets_id = vets.id  JOIN animals ON visits.animals_id = animals.id WHERE vets.name = 'William Tatcher' AND date BETWEEN '2020-04-01' AND '2020-08-30';
```

Comment:
- [ ] Ouch! this is a close one but for the query at line `62` the vet's name should be "Stephanie Mendez" as specified in the project requirements.

Reply:
Fixed.